client/state/ui/full-height-layout/actions.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { FULL_HEIGHT_LAYOUT_ON, FULL_HEIGHT_LAYOUT_OFF } from 'state/action-types';
+
+export function setFullHeightLayout() {
+	return dispatch =>
+		dispatch( {
+			type: FULL_HEIGHT_LAYOUT_ON,
+		} );
+}
```

Comment:
Why is this action a thunk?

Both actions can be merged into one with a boolean parameter.

Reply:
This is another case where my React understanding wasn't up to scratch.